### PR TITLE
Add EFS IAM policy to Kubernetes workers

### PR DESCRIPTION
**Pull request by giorgiosironi**
Allows EFS usage, to share volumes between containers.

### CODE REVIEW

File: src/buildercore/terraform.py [hunk 1]
```diff
@@ -808,6 +808,32 @@ def _render_eks_workers_role(context, template):
...
+            'name': '%s--AmazonEFSKubernetes' % context['stackname'],
+            'path': '/',
+            'description': 'Allows management of EFS resources',
+            'policy': json.dumps({
+                "Version": "2012-10-17",
+                "Statement": [
+                    {
+                        "Effect": "Allow",
+                        "Action": [
+                            "elasticfilesystem:*",
```
Comment (giorgiosironi) [inline src/buildercore/terraform.py]: we can possibly be more specific here, what `Action`s are required? `Resource` is limited anyway to the AWS account and nothing else uses EFS, so not much to do on that field

**Comment by giorgiosironi** [comment]
Already applied to `kubernetes-aws--test`.

File: src/buildercore/terraform.py (hunk 1 above)
Comment (tayowonibi) [reply src/buildercore/terraform.py]: from https://docs.aws.amazon.com/efs/latest/ug/efs-api-permissions-ref.html, i think these permissions below should be ok

elasticfilesystem:DescribeFileSystems 
elasticfilesystem:DescribeMountTargets
elasticfilesystem:DescribeMountTargetSecurityGroups
elasticfilesystem:DescribeTags

**Comment by giorgiosironi** [comment]
Would like to know if we can assign fewer permissions than `elasticfilesystem:*` (only selected actions)? But not a blocker.

**Comment by tayowonibi** [comment]
> Would like to know if we can assign fewer permissions than `elasticfilesystem:*` (only selected actions)? But not a blocker.


https://github.com/elifesciences/builder/pull/573#discussion_r336468558

File: src/buildercore/terraform.py (hunk 1 above)
Comment (tayowonibi) [reply src/buildercore/terraform.py]: yes

**Comment by giorgiosironi** [comment]
Updated and reapplied.

**Pull request merged**